heroku/cli
# migrate(W-14169349): spaces: Upgrade trusted-ips:add

This PR moves `spaces:trusted-ips:add` to the cli and migrates to TS

### Test Instructions
1. checkout this branch and build
2. run `./bin/run/ spaces:trusted-ips:add THE_IP_ADDRESS --space YOUR_SPACE`
3. Verify the output is the same as the original.
